code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -283,12 +283,24 @@
 	desc = "A dart for use in shotguns. Can be injected with up to 30 units of any chemical."
 	icon_state = "cshell"
 	projectile_type = /obj/item/projectile/bullet/dart
+	var/reagent_amount = 30
+	var/reagent_react = TRUE
+
+/obj/item/ammo_casing/shotgun/dart/noreact
+	name = "cryostasis shotgun dart"
+	desc = "A dart for use in shotguns, using similar technolgoy as cryostatis beakers to keep internal reagents from reacting. Can be injected with up to 10 units of any chemical."
+	icon_state = "cnrshell"
+	reagent_amount = 10
+	reagent_react = FALSE
 
 /obj/item/ammo_casing/shotgun/dart/Initialize()
 	. = ..()
 	container_type |= OPENCONTAINER
-	create_reagents(30)
-	reagents.set_reacting(TRUE)
+	create_reagents(reagent_amount)
+	if(reagent_react)
```

Comment:
You are guaranteed to always have the var at least. `reagents.set_reacting(reagent_react)` please, skip the if.

Reply:
do the needful